### DISCUSSION TITLE
Uploads fail on macOS 15.0.1

### DISCUSSION THREAD

**Discussion by Nina-Menssen** [iterate-ch/cyberduck#16415] [Q&A]
This ticket system is to report bugs and feature requests. For support, visit the [help page](https://cyberduck.io/help/) first.

Uploads fail with macOS Sequoia 15.0.1

**To Reproduce**
1. Open a server (tried with external sftp server (port 22) and with a local NAS using FTP (port 21)
2. Drag a file larger than 1 GB to destination
3. Transfer begins and runs for a while
4. After quite some transferred data (tried 10 times, it happened after 300 MB to 1.1 GB) the transmission stops with error dialog displaying "Stream closed" (sftp) or "Broken pipe. The connection attempt was rejected. The server may be down, or your network may not be properly configured." (ftp).

Of course I would expect the uploads to succeed. I am using both server destinations since years and without any problem, always with CyberDuck (registered user). I know there is enough free space on these volumes. And it worked until I upgraded from macOS 15.0 to 15.0.1 yesterday. I also restarted my Mac (MacBook Pro 16" M3,)

**Screenshots**

macOS 15.0.1, language German
CyberDuck Version 9.0.2 (42108)

[cyberduck.log](https://github.com/user-attachments/files/17261199/cyberduck.log)
<img width="837" alt="Bildschirmfoto 2024-10-04 um 17 45 31" src="https://github.com/user-attachments/assets/352f63f7-4e53-4fa9-9922-ec9396404cd7">
<img width="837" alt="Bildschirmfoto 2024-10-04 um 18 04 08" src="https://github.com/user-attachments/assets/99756dd3-be24-42d7-9bbf-cc72477c5e32">
<img width="483" alt="Bildschirmfoto 2024-10-04 um 17 58 49" src="https://github.com/user-attachments/assets/de8cfd18-a3bf-4fe8-b183-365c6f535ec0">

**Comment by JeromeNelsonC** [comment]
Same issue here, but with Gdrive. On macOS 15.1 Beta (24B5070a)

**Comment by Nina-Menssen** [comment]
Retested with macOS 15.1 RC. No more problem with multiple large uploads (3 transfers with > 5 GB, one with 20 GB). Seems to have been one of the connectivity problems of macOS 15.0.x
